### ISSUE TITLE
CC-PVDZ calculation fails

### ISSUE THREAD

**Issue by vwcruzeiro** [merzlab/QUICK#34]
The following calculation fails with an error "Segmentation fault - invalid memory reference" with `quick`, but the same calculation passes with `quick.cuda`:
```
B3LYP BASIS=CC-PVDZ SCF=200 CHARGE=0 MULT=1 GRADIENT DIPOLE
 
 H         5.4269280037321330       5.8668001759742197       1.9440424210060896 
 C         5.2778863555062401       6.1653696715003079       0.8517883878991734 
 H         5.9665759620204497       6.8917325525739459       0.4522474046750474 
 H         4.4104954720977245       6.6466044949751728       0.9546987362515305 
 C         5.1169729983679595       4.9326993945007960      -0.0230474591437522 
 O         4.1460063240795630       4.6757957711199207      -0.6787269435924508 
 N         6.1595314449254053       4.0934234816143507       0.1359075710503025 
 H         6.7649147303327437       4.3215394902951143       0.9156516725231513 
 C         6.2481130228304949       2.8038163311913320      -0.5367658543372317 
 H         5.5175447980454653       2.6865231194871195      -1.3190682664333815 
 H         7.1867685623498705       2.6684759996505929      -1.0046972369643465 
 H         6.1319761352814020       2.0088922338612880       0.2503952716044978
```

**Comment by agoetz** [comment]
Bug has been fixed in PR #35 

@Madu86 can you add this to the test suite?

**Comment by agoetz** [comment]
Test has been added in PR #37 
Closing this PR.
